### PR TITLE
Adding SQLAlchemy classes

### PR DESCRIPTION
**Pull request by meg25ellele**

### CODE REVIEW

File: orchestrator/dags/sql_alchemy_classes.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    unit_post = Column(Integer, primary_key=True)
+    full_name = Column(String)
+    stop_longitude = Column(Float)
+    stop_latitude = Column(Float)
+    street = Column(String)
+    unit = Column(String)
+    post = Column(String)
+    is_depot = Column(Integer)
+    create_at = Column(DateTime)
```
Comment (mgierm) [inline orchestrator/dags/sql_alchemy_classes.py]:
```suggestion
    created_at = Column(DateTime)
```

File: orchestrator/dags/sql_alchemy_classes.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    _tablename_ = "stops"
+
+    unit_post = Column(Integer, primary_key=True)
+    full_name = Column(String)
+    stop_longitude = Column(Float)
+    stop_latitude = Column(Float)
+    street = Column(String)
+    unit = Column(String)
+    post = Column(String)
+    is_depot = Column(Integer)
```
Comment (mgierm) [inline orchestrator/dags/sql_alchemy_classes.py]: `is_depot` probably should be a `Bool`.

File: orchestrator/dags/sql_alchemy_classes.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    create_at = Column(DateTime)
+
+    def __init__(self, unit_post: int, full_name: str, stop_longitude: float, stop_latitude: float, street: str,
```
Comment (mgierm) [inline orchestrator/dags/sql_alchemy_classes.py]: These `__init__` methods are not necessary. All of them could be removed.

File: orchestrator/dags/sql_alchemy_classes.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
+from sqlalchemy import Column, String, Integer, DateTime, ForeignKey, Float
+from sqlalchemy.ext.declarative import declarative_base
+from datetime import datetime
+
+base = declarative_base()
```
Comment (mgierm) [inline orchestrator/dags/sql_alchemy_classes.py]:
```suggestion
Base = declarative_base()
```

`declarative_base` is a Class constructor, so the name `Base` should be named like a class.

**Review by mgierm — CHANGES_REQUESTED** [review]
It's a good piece of work in general :+1: I added few notices, most importat is that we don't need the `__init__` methods, so you can remove them without worries. 

One point which I just noticed - one week ago the new version of SQLAlchemy was released - 2.0. It changes the approach of declaring class fields, take a look -> https://docs.sqlalchemy.org/en/20/orm/mapping_styles.html#orm-declarative-mapping

But I think we can stick to the previous version 1.4.x. (just for learning purposes).

**Comment by meg25ellele** [comment]
@mgierm @klepper24 @Maryl12342 I added some changes related to Mikolaj comment

**Review by mgierm — APPROVED** [review]
:+1:

**Review by klepper24 — APPROVED** [review]

**Pull request merged**